v63003/u_inref/gtm8844.csh

```diff
@@ -0,0 +1,116 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+# Lack permissions to write to the $ydb_dist directory, so we are creating a temp directory
+# and copying all files from $ydb_dist to temp, changing the ydb_dist environment variable
+# and putting restrict.txt in temp
+
+mkdir temp
+cp $ydb_dist/* temp/ >>& not_copied.out
+set ydb_dist0=$ydb_dist
+setenv ydb_dist temp
+cat > $ydb_dist/restrict.txt << EOF
+ZHALT
+HALT
+EOF
+chmod -w $ydb_dist/restrict.txt
+
+# Creating trigger file
+cat > trigger.txt << EOF
++^X -name=triggerx -command=set -xecute="do triggerfnx^gtm8844"
++^Y -name=triggery -command=set -xecute="do triggerfny^gtm8844"
+EOF
+
+echo "# Running functions that violate our restrictions"
+echo "# Expect two GTM_FATAL files at the bottom of the reference file"
+echo "# -------------------------------------------------------------------------------------------"
+echo "# Two consecutive halts"
+$ydb_dist/mumps -run haltfn^gtm8844
+ls *FATAL*
+rm *FATAL*
+echo "# -------------------------------------------------------------------------------------------"
+echo "# Two consecutive zhalts"
+$ydb_dist/mumps -run zhaltfn^gtm8844
+ls *FATAL*
+rm *FATAL*
+echo "# -------------------------------------------------------------------------------------------"
+echo "# Confirming ZHALT,ZGOTO 0 produces an error in the shell"
+$ydb_dist/mumps -run zgotofn^gtm8844
+echo "Status After ZGOTOFN=$status"
+if ($status) then
+	echo "Error Detected"
+endif
+echo "# -------------------------------------------------------------------------------------------"
+$MULTISITE_REPLIC_PREPARE 2
+setenv gtm_test_repl_norepl 1
+$gtm_tst/com/dbcreate.csh mumps 9 >>& dbcreate.out
+$MSR START INST1 INST2 >>& MSRStart.out
+
+# Creating initial values for global variables X and Y, uploading triggers
+$MSR RUN INST1 '$ydb_dist/mumps -run ^%XCMD "set ^X=0  set ^Y=0"'
+$MUPIP trigger -triggerfile=trigger.txt
+
+echo ""
+
+echo "# Test for ZHALT in a trigger"
```

Comment:
Is it testing that ZHALT in a trigger takes down the update process? If so, can this line mention that more descriptive scenario.